### PR TITLE
Adding skeleton file for Palo Alto

### PR DESCRIPTION
**Pull request by amahpour**
Adding things to do for Palo Alto

@mmoralJPL, is this good?

### CODE REVIEW

**Comment by mmoralJPL** [comment]
This code will never work, try again please!!!!! :smiley:

**Comment by amahpour** [comment]
This pull request fixes #10

File: amahpour-paloalto.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Palo Alto
```
Comment (mmoralJPL) [inline amahpour-paloalto.md:1]: Nice city!

File: amahpour-paloalto.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Places to eat
+
```
Comment (mmoralJPL) [inline amahpour-paloalto.md:5]: What do you recommend?

File: amahpour-paloalto.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+### Things to do
+
+### Cool sites
```
Comment (loriNakamura) [inline amahpour-paloalto.md:9]: Great!  Please explain!

File: amahpour-paloalto.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Palo Alto
+
+### Places to eat
+
+### Things to do
+
```
Comment (mmoralJPL) [inline amahpour-paloalto.md:8]: How close are you to the beach?

**Review by mmoralJPL — COMMENTED** [review]
Left some comments check them out.

File: amahpour-paloalto.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Palo Alto
+
+### Places to eat
```
Comment (loriNakamura) [inline amahpour-paloalto.md:3]: Any vegetarian places?

**Review by loriNakamura — CHANGES_REQUESTED** [review]
Please give me details on this awesome place!

**Review by amahpour — COMMENTED** [review]
Looks good

File: amahpour-paloalto.md (hunk 5 above)
Comment (amahpour) [reply amahpour-paloalto.md:3]: Added a bagel place

**Review by loriNakamura — APPROVED** [review]

**Pull request merged**